### ISSUE TITLE
Date format in Chinese is incorrect

### ISSUE THREAD

**Issue by think-in-universe** [near/near-explorer#1118]
### Description

The Chinese date format `第 1010日, 2022, 下午6:44:45` is wrong. 

The correct one should be `10月10日, 2022, 下午6:44:45`

This is probably caused by change made [here](https://github.com/near/near-explorer/pull/1039/files#diff-67a51ac485d4380ebb2428a29fc08e3cbe0659ec7dae9fce48081e6e2946146d)

### Example

https://explorer.testnet.near.org/accounts/v4-lfg.community-genesis2.bhc8521.testnet

<img width="326" alt="image" src="https://user-images.githubusercontent.com/46699230/203107976-0df7785d-c78a-453e-b424-3447fa4b4913.png">

**Comment by frol** [comment]
@think-in-universe Would you be able to submit a PR to the code here: https://github.com/near/near-explorer/tree/master/frontend/src/libraries/locales? 🙏

**Comment by think-in-universe** [comment]
@frol  yeah. I'm working on this today.

**Comment by think-in-universe** [comment]
@frol Is there any way to start the frontend on testnet without launching the backend server? 

I have trouble to start the backend due to issues

```
✨ backend % npm run dev:mainnet

> near-explorer-backend@0.0.1 dev:mainnet
> dotenv -c -e ../mainnet.env -- npm run dev


> near-explorer-backend@0.0.1 dev
> nodemon src/index.ts

[nodemon] 2.0.20
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: ts,json
[nodemon] starting `ts-node src/index.ts`
/Users/robort/workspace/blockchain/projects/near/projects/toolbox/near-explorer/backend/node_modules/ts-node/src/index.ts:859
    return new TSError(diagnosticText, diagnosticCodes, diagnostics);
           ^
TSError: ⨯ Unable to compile TypeScript:
src/router/index.ts:17:20 - error TS2345: Argument of type 'Router<Context, Context, {}, Record<"protocolVersion", Procedure<Context, Context, {}, undefined, undefined, number, unknown, number>> & Record<...>, {}, {}, DefaultErrorShape>' is not assignable to parameter of type 'Router<unknown, any, {}, any, any, any, any>'.
  The types of '_def.middlewares' are incompatible between these types.
    Type 'MiddlewareFunction<Context, Context, {}>[]' is not assignable to type 'MiddlewareFunction<unknown, any, {}>[]'.
      Type 'MiddlewareFunction<Context, Context, {}>' is not assignable to type 'MiddlewareFunction<unknown, any, {}>'.
        Type 'unknown' is not assignable to type 'Context'.

17   .merge("utils.", utilsRouter)
                      ~~~~~~~~~~~
src/router/index.ts:18:20 - error TS2345: Argument of type 'Router<Context, Context, {}, Record<"latestCirculatingSupply", Procedure<Context, Context, {}, undefined, undefined, Selection<TableExpressionDatabase<ModelTypeMap, "aggregated__circulating_supply", "aggregated__circulating_supply">, "aggregated__circulating_supply", "circulating_tokens_supply as supply" | "computed...' is not assignable to parameter of type 'Router<unknown, any, {}, any, any, any, any>'.
  The types of '_def.middlewares' are incompatible between these types.
    Type 'MiddlewareFunction<Context, Context, {}>[]' is not assignable to type 'MiddlewareFunction<unknown, any, {}>[]'.

18   .merge("stats.", statsRouter)
                      ~~~~~~~~~~~
src/router/index.ts:19:20 - error TS2345: Argument of type 'Router<Context, Context, {}, Record<"final", Procedure<Context, Context, {}, { source: "rpc"; } | { source: "indexer"; }, { source: "rpc"; } | { source: "indexer"; }, Selection<...> | { ...; }, unknown, Selection<...> | { ...; }>> & Record<...> & Record<...> & Record<...>, {}, {}, DefaultErrorShape>' is not assignable to parameter of type 'Router<unknown, any, {}, any, any, any, any>'.
  The types of '_def.middlewares' are incompatible between these types.
    Type 'MiddlewareFunction<Context, Context, {}>[]' is not assignable to type 'MiddlewareFunction<unknown, any, {}>[]'.

19   .merge("block.", blockRouter)
                      ~~~~~~~~~~~
src/router/index.ts:20:26 - error TS2345: Argument of type 'Router<Context, Context, {}, Record<"byHashOld", Procedure<Context, Context, {}, { hash: string; }, { hash: string; }, { hash: string; signerId: string; receiverId: string; ... 5 more ...; outcome: { ...; }; } | null, unknown, { ...; } | null>> & ... 5 more ... & Record<...>, {}, {}, DefaultErrorShape>' is not assignable to parameter of type 'Router<unknown, any, {}, any, any, any, any>'.
  The types of '_def.middlewares' are incompatible between these types.
    Type 'MiddlewareFunction<Context, Context, {}>[]' is not assignable to type 'MiddlewareFunction<unknown, any, {}>[]'.

20   .merge("transaction.", transactionRouter)
                            ~~~~~~~~~~~~~~~~~
src/router/index.ts:21:22 - error TS2345: Argument of type 'Router<Context, Context, {}, Record<"listIncludedByBlockHash", Procedure<Context, Context, {}, { blockHash: string; }, { blockHash: string; }, Receipt[], unknown, Receipt[]>> & Record<...> & Record<...>, {}, {}, DefaultErrorShape>' is not assignable to parameter of type 'Router<unknown, any, {}, any, any, any, any>'.
  The types of '_def.middlewares' are incompatible between these types.
    Type 'MiddlewareFunction<Context, Context, {}>[]' is not assignable to type 'MiddlewareFunction<unknown, any, {}>[]'.

21   .merge("receipt.", receiptRouter)
                        ~~~~~~~~~~~~~
src/router/index.ts:22:22 - error TS2345: Argument of type 'Router<Context, Context, {}, Record<"byIdOld", Procedure<Context, Context, {}, { id: string; }, { id: string; }, { details: { storageUsage: number; stakedBalance: string; nonStakedBalance: string; lockupAccountId: string | undefined; } | null; accountId: string; created: { ...; } | undefined; deleted: { ...; } | und...' is not assignable to parameter of type 'Router<unknown, any, {}, any, any, any, any>'.
  The types of '_def.middlewares' are incompatible between these types.
    Type 'MiddlewareFunction<Context, Context, {}>[]' is not assignable to type 'MiddlewareFunction<unknown, any, {}>[]'.

22   .merge("account.", accountRouter)
                        ~~~~~~~~~~~~~
src/router/index.ts:23:23 - error TS2345: Argument of type 'Router<Context, Context, {}, Record<"byId", Procedure<Context, Context, {}, { id: string; }, { id: string; }, { codeHash: string; locked: boolean; transactionHash?: undefined; timestamp?: undefined; } | { ...; } | null, unknown, { ...; } | ... 1 more ... | null>>, {}, {}, DefaultErrorShape>' is not assignable to parameter of type 'Router<unknown, any, {}, any, any, any, any>'.
  The types of '_def.middlewares' are incompatible between these types.
    Type 'MiddlewareFunction<Context, Context, {}>[]' is not assignable to type 'MiddlewareFunction<unknown, any, {}>[]'.

23   .merge("contract.", contractRouter)
                         ~~~~~~~~~~~~~~
src/router/index.ts:24:24 - error TS2345: Argument of type 'Router<Context, Context, {}, {}, Record<"upsert", Procedure<Context, Context, {}, { signature?: string | undefined; ipAddress: string; agent: { name: string; version: string; build: string; }; system: { ...; }; chain: { ...; }; }, { ...; }, void, unknown, void>>, {}, DefaultErrorShape>' is not assignable to parameter of type 'Router<unknown, any, {}, any, any, any, any>'.
  The types of '_def.middlewares' are incompatible between these types.
    Type 'MiddlewareFunction<Context, Context, {}>[]' is not assignable to type 'MiddlewareFunction<unknown, any, {}>[]'.

24   .merge("telemetry.", telemetryRouter)
                          ~~~~~~~~~~~~~~~
src/router/index.ts:25:29 - error TS2345: Argument of type 'Router<Context, Context, {}, Record<"amount", Procedure<Context, Context, {}, undefined, undefined, number, unknown, number>> & Record<...>, {}, {}, DefaultErrorShape>' is not assignable to parameter of type 'Router<unknown, any, {}, any, any, any, any>'.
  The types of '_def.middlewares' are incompatible between these types.
    Type 'MiddlewareFunction<Context, Context, {}>[]' is not assignable to type 'MiddlewareFunction<unknown, any, {}>[]'.

25   .merge("fungibleTokens.", fungibleTokensRouter);
                               ~~~~~~~~~~~~~~~~~~~~

    at createTSError (/Users/robort/workspace/blockchain/projects/near/projects/toolbox/near-explorer/backend/node_modules/ts-node/src/index.ts:859:12)
    at reportTSError (/Users/robort/workspace/blockchain/projects/near/projects/toolbox/near-explorer/backend/node_modules/ts-node/src/index.ts:863:19)
    at getOutput (/Users/robort/workspace/blockchain/projects/near/projects/toolbox/near-explorer/backend/node_modules/ts-node/src/index.ts:1077:36)
    at Object.compile (/Users/robort/workspace/blockchain/projects/near/projects/toolbox/near-explorer/backend/node_modules/ts-node/src/index.ts:1433:41)
    at Module.m._compile (/Users/robort/workspace/blockchain/projects/near/projects/toolbox/near-explorer/backend/node_modules/ts-node/src/index.ts:1617:30)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Object.require.extensions.<computed> [as .ts] (/Users/robort/workspace/blockchain/projects/near/projects/toolbox/near-explorer/backend/node_modules/ts-node/src/index.ts:1621:12)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  diagnosticCodes: [
    2345, 2345, 2345,
    2345, 2345, 2345,
    2345, 2345, 2345
  ]
}
[nodemon] app crashed - waiting for file changes before starting...
```

**Comment by luixo** [comment]
```
v16.17.0
```
We should add `.nvmrc` though to keep it in line.

Can you please remove all `node_modules` from the project or run `npm ci` (I'm not how that works with monorepos)? Index one and also in `common` / `frontend` / `backend` dirs.

**Comment by think-in-universe** [comment]
> Can you please remove all `node_modules` from the project or run `npm ci` (I'm not how that works with monorepos)? Index one and also in `common` / `frontend` / `backend` dirs.

It doesn't work. But I can successfully run the backend with Docker.

I have fixed the issue here: #1120    Please take a look.
